test/px-data-table-test.js

```diff
@@ -1,6 +1,4 @@
-var table1Fixture, table2Fixture, table3Fixture, table4Fixture, table5Fixture, filtertest, resetDataFixture, 
-  additionalDataFixture, updateSelectFixture, remoteDataFixture1, remoteDataFixture2, remoteDataFixture3, 
-  remoteDataFilteringFixture1, remoteDataFilteringFixture2, greedyHeightWithScroll;
+var table1Fixture, table2Fixture, table3Fixture, table4Fixture, table5Fixture, filtertest, resetDataFixture, additionalDataFixture, updateSelectFixture, remoteDataFixture1, remoteDataFixture2, remoteDataFixture3,remoteDataFilteringFixture1, remoteDataFilteringFixture2, greedyHeightWithScrollFixture, clientSmartRowSelectionWithLargeHeightFixture, serverSmartRowSelectionWithLargeHeightFixture;
```

Comment:
Keep the line breaks as they had been added intentionally. Make sure to not remove any variables. Ensure there is always a space after a comma.